README.md

```diff
@@ -19,11 +19,32 @@ Pairmotron is an app powered by [Phoenix](http://www.phoenixframework.org/) that
 ```
 export PG_USER="example_username"
 export PG_PASSWORD="example_password"
+export PG_HOST="localhost"
 ```
   * Create and migrate your database with `mix ecto.create && mix ecto.migrate`
   * Install Node.js dependencies with `npm install`
   * Start Phoenix endpoint with `mix phoenix.server`
 
+## Developing and Testing with Docker
+
+Using this model of development allows you to not install anything other
+than docker.
+
+### Running it
+  * docker-compose up -d web
+  * docker-compose exec web mix deps.get
+  * docker-compose exec web mix ecto.create 
+  * docker-compose exec web mix ecto.migrate
+  * docker-compose exec web npm install
+  * docker-compose restart web
+
+### Run the Tests in Docker
+  * docker-compose run -e "MIX_ENV=test" web mix test 
+  * Only test a specific test file
+    * docker-compose run -e "MIX_ENV=test" web mix test/controllers/group_controller_test.exs
```

Comment:
Typo in `web mix test/controllers/group_controller_test.exs`. Should be `web mix 
test test/controllers/group_controller_test.exs`